client/my-sites/email/inbox/mailbox-selection-list/index.jsx

```diff
@@ -137,7 +137,7 @@ const MailboxItems = ( { mailboxes } ) => {
 				align="center"
 				brandFont
 				className="mailbox-selection-list__header"
-				headerText={ translate( 'Welcome to Inbox!' ) }
+				headerText={ translate( 'My Mailboxes!' ) }
```

Comment:
>Is that intentional or just a leftover from the previous copy?

Good eye. I think it's just a leftover from the previous copy :) I agree it's odd since we removed "Welcome to". @fredrikekelund Could you help create a quick PR to remove the `!`?

Reply:
Sure, I'll do that!